addons/account_edi_ubl_cii/models/res_partner.py

```diff
@@ -135,47 +135,58 @@ def _check_peppol_fields(self):
 
     @api.depends('country_code')
     def _compute_ubl_cii_format(self):
+        format_mapping = {
+            'DE': 'xrechnung',
+            'AU': 'ubl_a_nz',
+            'NZ': 'ubl_a_nz',
+            'NL': 'nlcius',
+            'FR': 'facturx',
+            'SG': 'ubl_sg',
```

Comment:
Should be this be a real function instead of an inner one? Then, if a module adds a new entry in the mapping, it will work without overriding `_compute_ubl_cii_format` :thinking: